fdbrpc/sim2.actor.cpp

```diff
@@ -2305,6 +2311,60 @@ class Sim2 final : public ISimulator, public INetworkConnections {
 		dieWhenTerminate = false;
 	}
 
+	int forkSearch(const char* context = "") override {
+#if defined(_WIN32)
+		printf("start fork...");
```

Comment:
Wouldn't this print only for Windows? I thought we were only planning to support *nix 🤔